packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -172,6 +149,32 @@ export const workflowsRouter = createProtectedRouter()
         },
       });
 
+      if (deleted.count === 1) {
```

Comment:
Delete the scheduled reminders only if the workflow was successfully deleted

Reply:
Do we have onCascade deletion on prisma schema? In theory if we have this, we only need to delete the workflow and all reminders will be deleted without any extra logic

Reply:
I'm fairly certain we did add this for user account deletion

Reply:
@zomars True, we already have onCascade deletion in the Prisma schema defined. Didn't think about that, but we still need to make sure to delete the scheduled reminders on SendGrid and Twilio with deleteScheduledEmailReminder and deleteScheduledSMSReminder. I'll made some changes here: https://github.com/calcom/cal.com/pull/3661/commits/8d4cc753675efd4675af5b527b69e83354a14304